script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -109,14 +109,14 @@ Even if I don't feel anxious about it, the mere thought of returning to such an
 The train arrives at a large station and stops for several minutes.
 Since this train is stopping at every station, I'll have to wait for the next rapid train to pass by.
 <Page30>
-There is no one else on the platform.
+No one's there on the platform.//I think the line is meant to say that there's actually no one on the platform.
 At this time of day, I would expect to see a few businessmen in suits, but this morning is particularly quiet.
 <Page31>
 An express train is passing by on the next track.
 'Thank you for waiting. The next stop is―――'
 I hear a familiar announcement.
 The automatic door closes using pneumatic pressure.
-There are only four stops left until Souya Station, where my high school is located.
+There are only four more stops left until Souya Station, where my high school is located.
```

Comment:
I feel like having both 'more' and 'left' is redundant. I would revert. Maybe change 'until' to 'to'.

> There are only four stops left to Souya Station,

Reply:
Yeah, just one word is enough. I would just revert.